cirq-core/cirq/ops/control_values.py

```diff
@@ -22,10 +22,21 @@
 
 
 class AbstractControlValues(abc.ABC):
-    """AbstractControlValues is an abstract immutable data class.
-
-    AbstractControlValues defines an API for control values and implements
-    functions common to all implementations (e.g.  comparison).
+    """AbstractControlValues is an abstract class that defines an API for
+
+       control values and implements functions common to all implementations
+       (e.g.  comparison).
```

Comment:
Multi-line docstrings should follow the convention of a single one-line summary, followed by a blank line and then a detailed description (See https://peps.python.org/pep-0257/#multi-line-docstrings). 

```suggestion
    """Abstract base class defining the API for control values.
    
    `AbstractControlValues` is an abstract class that defines the API for control values 
    and implements functions common to all implementations (e.g.  comparison).
```